cpp/tests/transform/row_bit_count_test.cu

```diff
@@ -292,7 +294,8 @@ std::pair<std::unique_ptr<cudf::column>, std::unique_ptr<cudf::column>> build_st
   auto size_iter =
     cudf::detail::make_counting_transform_iterator(0, [&strings, &struct_validity](int i) {
       return (sizeof(float) * CHAR_BIT) + 1 + (sizeof(int16_t) * CHAR_BIT) + 1 +
-             (static_cast<cudf::size_type>(strings[i].size()) * CHAR_BIT) +
+             ((struct_validity[i] ? static_cast<cudf::size_type>(strings[i].size()) : 0) *
+              CHAR_BIT) +
              (sizeof(cudf::offset_type) * CHAR_BIT) + 1 + 1;
```

Comment:
Perhaps these values should be hardcoded instead of calculated. It seems to be duplicating code logic for `cudf::row_bit_count()`. This can be considered outside the scope of this PR.

Reply:
I thought about that when fixing the tests in this file. After a couple of times I wrongly calculated the values, I gave up and decided to just continue using the code :smile:

Reply:
More tech debt then.